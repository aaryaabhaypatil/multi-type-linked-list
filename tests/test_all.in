NEW 3
Aw
e
A
REMOVE 0
