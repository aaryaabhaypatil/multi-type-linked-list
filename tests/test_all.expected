List 0: Aw -> e -> A
List 0 has been removed.

Number of Lists: 0
